src/shims/dir.props

```diff
@@ -25,6 +26,12 @@
   </ItemGroup>
 
   <PropertyGroup>
-  	<GenFacadesOutputPath>$(BaseIntermediateOutputPath)/shims/facades/</GenFacadesOutputPath>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
Do they have to go in the Ref of both packages as well? I would suppose they would only be used at runtime, in which case we can set `IsNETCoreAppRef` and `IsUAPRef` to false.